<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product Administration</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="productsTab">
      <attribute name="title">
       <string>Product Management</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelCode">
           <property name="text">
            <string>Code:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditCode">
           <property name="placeholderText">
            <string>Enter product code</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditName">
           <property name="placeholderText">
            <string>Enter product name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelPrice">
           <property name="text">
            <string>Price:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditPrice">
           <property name="placeholderText">
            <string>Enter product price</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="buttonAdd">
           <property name="text">
            <string>Add Product</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonUpdate">
           <property name="text">
            <string>Update Product</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDelete">
           <property name="text">
            <string>Delete Product</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="adminsTab">
      <attribute name="title">
       <string>Admin Management</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidgetAdmins">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Username</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="adminFormLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelAdminUsername">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditAdminUsername">
           <property name="placeholderText">
            <string>Enter new admin username</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelAdminPassword">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditAdminPassword">
           <property name="placeholderText">
            <string>Enter new admin password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="adminButtonLayout">
         <item>
          <widget class="QPushButton" name="buttonAddAdmin">
           <property name="text">
            <string>Add Admin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonDeleteAdmin">
           <property name="text">
            <string>Delete Selected Admin</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>